doc/changes.md

```diff
@@ -1,5 +1,54 @@
 # Changelog
 
+## Release v0.4.1 (2021/08/07)
+
+[![Digital Object Identifier for PyGMT v0.4.1](https://zenodo.org/badge/DOI/10.5281/zenodo.5162003.svg)](https://doi.org/10.5281/zenodo.5162003)
+
+### Highlights
+
+* 🎉 **Patch release with multiple gallery examples** 🎉
+* Change references from "master" branch to "main" branch ([#1360](https://github.com/GenericMappingTools/pygmt/pull/1360))
```

Comment:
> Change references from "master" branch to "main" branch

Change the above sentence to:

> Change the default branch name from "master" to "main" to increase inclusivity

Reply:
Done in https://github.com/GenericMappingTools/pygmt/pull/1414/commits/5426bbef085c3de95b2401e457992230ecfe5aee